arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -135,46 +164,55 @@
             "$ref": "#/parameters/apiVersionParameter"
           },
           {
-            "name": "resourceGroupName",
-            "description": "Name of the resource group within the Azure subscription.",
-            "in": "path",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
-            "name": "mediaServiceName",
-            "description": "Name of the Media Service.",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "pattern": "^[a-z0-9]",
-            "minLength": 3,
-            "maxLength": 24
+            "$ref": "#/parameters/mediaResourceNameParameter"
           }
-        ]
-      },
-      "put": {
-        "summary": "Create a Media Service.",
-        "operationId": "MediaService_Create",
+        ],
         "responses": {
           "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/MediaService"
-            }
-          },
-          "201": {
-            "description": "Created",
+            "description": "Success. Returns the details of the Media Services account.",
             "schema": {
               "$ref": "#/definitions/MediaService"
+            },
+            "examples" : {
+              "application/json": {
+                "name": "contosomedia",
+                "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Media/mediaservices/contosomedia",
+                "type": "Microsoft.Media/mediaservices",
+                "location": "West US",
+                "tags":null,
+                "properties":{  
+                  "apiEndpoints":[  
+                    {  
+                      "endpoint": "https://wamsbayclus001rest-hs.cloudapp.net/api/",
+                      "majorVersion": "2"
+                    }
+                  ],
+                  "storageAccounts":[  
+                    {  
+                      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-media/providers/Microsoft.Storage/storageAccounts/contosomedia",
+                      "isPrimary": true
+                    }
+                  ]
+                }
+              }  
             }
           },
           "default": {
-            "description": "Detailed error information.",
+            "description": "Get Media Services failed.",

```

Comment:
Would this be better as "Get Media Services failures.", or "Get Media Services that failed", or "Get Media Services operation failed." ?